docs: jira cloud documentation (#3494)
Unleash/unleash: website/docs/reference/integrations/jira-cloud-plugin-installation.md

---
title: Jira Cloud Integration - Installation
---

## Prerequisites

### Jira Cloud

- [Jira **Cloud**](https://www.atlassian.com/software/jira)

> For Jira Data Center, [see the Jira Server plugin](/integrations/jira_server_plugin_installation)

Author: 👍

### Unleash

- Unleash v4 or higher

## Required access levels

### Unleash

> You will need an Unleash admin to configure an admin API token. (Unleash Open Source)

Author: Good point, I'll try to come up with something better.

> For Unleash Enterprise we recommend [Service Tokens](https://docs.getunleash.io/how-to/how-to-create-service-accounts), this is also a requirement for integrating with Change Requests

Reviewer: I think this should link to the reference documentation instead. We should also link to change requests. The links in this suggestion have not been checked, so they may be wrong. 

```suggestion
For Unleash Enterprise we recommend using a [service account](../service-accounts.md). Service accounts are also required to integrate with [Change Requests](../change-requests.mdx)
```

### Jira

You will need a Jira admin user.

## Installation

If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.

> More to be added here after we can have screenshots and guides of the installation procedure

## Configuring the plugin {#configure-plugin}

After the plugin has been installed, each project's settings page in Jira will have a menu entry link called "Unleash Project Settings" under the "Apps" menu section.

> ![Jira Cloud - Project settings link](/img/jira_cloud_project_settings_link.png)

Reviewer: Can we improve this alt text? Without seeing this image, I have no idea what it is supposed to be. For people with vision impairments or with slow enough networks that images are impractical: what should they get out of the image?

The same goes for the rest of the alt texts in this PR.

---
title: Jira Cloud Integration - Installation
---

> 

Reviewer: Optional: you'll only need this if you use images with captions. (A lot of my sugggestions do)

```suggestion
import Figure from '@site/src/components/Figure/Figure.tsx'

```

Example of an image with a caption:
![image](https://user-images.githubusercontent.com/17786332/231954660-f7411f9f-4551-4811-8c62-4deb6d99e0d7.png)

## Prerequisites

### Jira Cloud

- [Jira **Cloud**](https://www.atlassian.com/software/jira)

For Jira Data Center, check out the [Jira Server plugin](jira-server-plugin-installation.md)

### Unleash

- Unleash v4 or higher

## Required access levels

### Unleash

You will need an Unleash admin user to configure the access tokens needed to connect the plugin to Unleash.

For Unleash Open Source this plugin requires the use of an [Admin API token](../api-tokens-and-client-keys.mdx) to connect.

For Unleash Enterprise we recommend using a [service account](../service-accounts.md). Service accounts are also required to integrate with [Change Requests](../change-requests.md)

### Jira

You will need a Jira admin user.

## Installation

If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.

> > More to be added here after we can have screenshots and guides of the installation procedure

Author: It was intended, but now that you mention it it does feel a bit `// TODO:` and can be taken care of in a better way process wise. will fix

> > More to be added here after we can have screenshots and guides of the installation procedure

Reviewer 2: I vote we remove this. We do need the review to complete but that's uncertain since it's on Atlassian timelines. I'd rather have our docs in a nice state than have confusing looking placeholders

> > More to be added here after we can have screenshots and guides of the installation procedure

Author: Removed!

## Configuring the plugin {#configure-plugin}

After the plugin has been installed, each project's settings page in Jira will have a menu entry link called "Unleash Project Settings" under the "Apps" menu section.

> ![Jira Cloud - Link to Unleash Project Settings in Jira Project Settings Apps menu](/img/jira-cloud-project-settings-link.png)

Reviewer: Hmm, it feels like the main point of the image is to point to the project settings link, so let's see if we can do the same in text (without repeating the preceding paragraph).

So for a regular image:

```suggestion
![Jira Cloud project settings page with the apps menu open. The link to Unleash project settings is highlighted.](/img/jira-cloud-project-settings-link.png)
```

Author: Thank you, looks like a good improvement!

Following that link takes you to the "Unleash Project Settings" configuration page. This is where you specify the connection details (Unleash server URL and access token) for the Unleash server to be used with this particular project.

> ![Jira Cloud - Project settings page for the Unleash Jira Cloud Plugin](/img/jira-cloud-project-settings.png)

Reviewer: ```suggestion
![Jira Cloud: Unleash project settings. A form with inputs for Unleash URL and an Unleash auth token.](/img/jira-cloud-project-settings.png)
```
